ibis/clickhouse/client.py

```diff
@@ -2,69 +2,137 @@
 import pandas as pd
 
 import ibis.common as com
-import ibis.expr.operations as ops
-import ibis.expr.schema as sch
 import ibis.expr.types as ir
+import ibis.expr.schema as sch
+import ibis.expr.lineage as lin
+import ibis.expr.datatypes as dt
+import ibis.expr.operations as ops
 
 from ibis.config import options
 from ibis.compat import zip as czip
 from ibis.client import Query, Database, DatabaseEntity, SQLClient
-from ibis.clickhouse.compiler import build_ast, ClickhouseDialect
+from ibis.clickhouse.compiler import ClickhouseDialect, build_ast
 from ibis.util import log
 from ibis.sql.compiler import DDL
 
 from clickhouse_driver.client import Client as _DriverClient
 
-from .types import clickhouse_to_pandas, clickhouse_to_ibis
-
 
 fully_qualified_re = re.compile(r"(.*)\.(?:`(.*)`|(.*))")
 
 
+_clickhouse_dtypes = {
+    'Null': dt.Null,
+    'UInt8': dt.UInt8,
+    'UInt16': dt.UInt16,
+    'UInt32': dt.UInt32,
+    'UInt64': dt.UInt64,
+    'Int8': dt.Int8,
+    'Int16': dt.Int16,
+    'Int32': dt.Int32,
+    'Int64': dt.Int64,
+    'Float32': dt.Float32,
+    'Float64': dt.Float64,
+    'String': dt.String,
+    'FixedString': dt.String,
+    'Date': dt.Date,
+    'DateTime': dt.Timestamp
+}
+_ibis_dtypes = {v: k for k, v in _clickhouse_dtypes.items()}
+_ibis_dtypes[dt.String] = 'String'
+
+
+class ClickhouseDataType(object):
```

Comment:
Clickhouse doesn't depend on alchemy.

Reply:
I've created an issue for that https://github.com/mymarilyn/clickhouse-driver/issues/22, no reaction yet. I could refactor this piece of code once `clickhouse-driver` ships column specification objects instead of strings.

Reply:
Cool, that sounds good to me, can you create an issue on our side as well.